packages/migrate/src/index.ts

```diff
@@ -174,8 +175,8 @@ function runMigration(currentConfigPath: string, outputConfigPath: string): Prom
 export default function migrate(...args: string[]): void | Promise<void> {
     const filePaths = args;
     if (!filePaths.length) {
-        const errMsg = '\n ✖ Please specify a path to your webpack config \n ';
-        console.error(red(errMsg));
+        const errMsg = '\n ✖ Please specify a path to your webpack config';
+        logger.error(errMsg);
```

Comment:
Better - `logger.error('\n ✖ Please specify a path to your webpack config')`.

Inline unnecessary extra variables.